optimum/exporters/ipex/modeling_utils.py

```diff
@@ -256,53 +222,147 @@ def forward(
         past_key_value: Optional[Tuple[torch.Tensor]] = None,
         output_attentions: bool = False,
         use_cache: bool = False,
-        cache_position: Optional[torch.LongTensor] = None,
-        residual: Optional[torch.Tensor] = None,
```

Comment:
why removing ?

Reply:
Because we don't need these 2 parameters after refactoring, the related ops are moved to IPEXDecoderLayer.